phpstan/phpstan/1.0/phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 6
+    paths:
+        - bin/
+        - config/
+        - public/
```

Comment:
I'm not sure about `public/` and `bin/`, though I suppose they don't hurt.

Reply:
Same as https://github.com/symfony/recipes-contrib/blob/main/squizlabs/php_codesniffer/3.6/phpcs.xml.dist#L13